### PR TITLE
Fix: corrected termsOfService in additional info

### LINKED ISSUE #718: Is termsOfServiceUrl incorrect?
Visible in https://github.com/artsy/doppler, http://developers.artsy.net

https://online.swagger.io/validator/debug?url=https%3A%2F%2Fstagingapi.artsy.net%2Fapi%2Fdocs

screen shot 2018-10-29 at 10 45 30 am

### PR DESCRIPTION
**Pull request by dblock**
Closes #718

### CODE REVIEW

File: lib/grape-swagger/endpoint.rb [hunk 1]
```diff
@@ -43,12 +43,12 @@ def swagger_object(target_class, request, options)
...
       result = {
-        title:             infos[:title] || 'API title',
-        description:       infos[:description],
-        termsOfServiceUrl: infos[:terms_of_service_url],
-        contact:           contact_object(infos),
-        license:           license_object(infos),
-        version:           infos[:version]
+        title: infos[:title] || 'API title',
+        description: infos[:description],
+        termsOfService: infos[:terms_of_service_url],
```
Comment (LeFnord) [inline lib/grape-swagger/endpoint.rb]: wow, this is in since year, and nobody saw that 😉

Comment (dblock) [reply lib/grape-swagger/endpoint.rb]: probably nobody cares about TOS ;)

**Comment by LeFnord** [comment]
think it is better to mörge #717 first, then this one, what did you think?

File: .travis.yml [hunk 2]
```diff
@@ -37,3 +37,4 @@ matrix:
     - rvm: 2.3.7
     - rvm: ruby-head
     - rvm: jruby-head
+    - env: GRAPE_VERSION=HEAD
```
Comment (LeFnord) [inline .travis.yml]: think, this can be removed now

**Comment by dblock** [comment]
💚 @LeFnord

**Comment by LeFnord** [comment]
thanks @dblock

**Pull request merged**